libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnChooseProperty.schema

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
+    "$role": "union(Microsoft.ITriggerCondition)",
+    "title": "ChooseProperty Event",
+    "description": "This defines the actions to take when a ChooseProperty event is raised",
```

Comment:
Can you update the language around this to match the other Onxxx definitions?
    "title": "On choose property",
    "description": "Actions to perform when there is ambiguity between multiple properties.",